parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -479,7 +455,13 @@ contexts:
   name: bootstrap-context
 current-context: bootstrap-context
 EOF
-    else
+    fi
+
+    if [ "${ENABLE_SECURE_TLS_BOOTSTRAPPING}" == "true" ]; then
+        configureKubeletSecureTLSBootstrap
+    fi
+
+    if [ "${ENABLE_SECURE_TLS_BOOTSTRAPPING}" == "false" ] && [ "${ENABLE_TLS_BOOTSTRAPPING}" == "false" ]; then
```

Comment:
Is it possible that only the first variable is set to true, and we don't create this kubeconfig file to fallback to?

Reply:
ah `it's the only option available if we can't have kubelet bootstrap its own credential (which is determined by RP).` Makes sense, thank you for explaining. Lmk when you need approval.